src/Training.TruckWorld.Clone.Backend/TruckWorld.Infrastructure/Common/Notifications/Services/EmailTemplateService.cs

```diff
@@ -0,0 +1,64 @@
+﻿using FluentValidation;
+using Microsoft.EntityFrameworkCore;
+using System.Linq.Expressions;
+using TruckWorld.Application.Common.Models.Querying;
+using TruckWorld.Application.Common.Notifications.Services;
+using TruckWorld.Application.Common.Querying.Extensions;
+using TruckWorld.Domain.Entities;
+using TruckWorld.Domain.Enums;
+using TruckWorld.Persistence.Repositories.Interfaces;
+
+namespace TruckWorld.Infrastructure.Common.Notifications.Services;
+/// <summary>
+/// created crud methods for emailTemplate
+/// </summary>
+public class EmailTemplateService : IEmailTemplateService
+{
+    private readonly IEmailTemplateRepository _emailTemplateRepository;
+    private readonly IValidator<EmailTemplate> _emailTemplateValidator;
+
+    public EmailTemplateService(
+        IEmailTemplateRepository emailTemplateRepository,
+         IValidator<EmailTemplate> emailTemplateValidator)
+    {
+        _emailTemplateRepository = emailTemplateRepository;
+        _emailTemplateValidator = emailTemplateValidator;
+    }
+
+
+    public IQueryable<EmailTemplate> Get(Expression<Func<EmailTemplate, bool>>? predicate = default,
+        bool asNoTracking = false,
+        CancellationToken cancellationToken = default)
+        =>
+            _emailTemplateRepository.Get(predicate, asNoTracking);
+
```

Comment:
'=> _emailTemplateRepository.Get(predicate, asNoTracking);'

i think , this version is good bro )